modules/compose/compose_api.go

```diff
@@ -401,7 +426,11 @@ func (d *dockerCompose) Up(ctx context.Context, opts ...StackUpOption) error {
 		})
 	}
 
-	return errGrpWait.Wait()
+	if err := errGrpWait.Wait(); err != nil {
```

Comment:
question: I think this change is for readability, or I'm missing something else?

Reply:
Nope that was just me forgetting to add the wrap.